modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/java.util.txt

```diff
@@ -910,22 +910,22 @@ class java.util.PriorityQueue {
 }
 
 class java.util.Random {
-  ()
-  (long)
-  DoubleStream doubles(long)
-  DoubleStream doubles(long,double,double)
-  IntStream ints(long)
-  IntStream ints(long,int,int)
-  LongStream longs(long)
-  LongStream longs(long,long,long)
-  boolean nextBoolean()
-  void nextBytes(byte[])
-  double nextDouble()
-  float nextFloat()
-  double nextGaussian()
-  int nextInt()
-  int nextInt(int)
-  long nextLong()
+  () @nondeterministic
+  (long) @nondeterministic
```

Comment:
maybe the unseeded constructor is the only one that needs to be annotated?

Reply:
Yup.